### ISSUE TITLE
Add support for PostgreSQL

### ISSUE THREAD

**Issue by jirutka** [tembo-io/prometheus_fdw#6]
Can you please add support for PostgreSQL 16?

**Comment by Jayko001** [comment]
Yes. We intend to bump up the pgrx and supabase wrappers version in our next release.

**Comment by Jayko001** [comment]
Update: supabase has not released the wrappers v0.2 to crates yet. I have created an issue for it https://github.com/supabase/wrappers/issues/227. Will provide an update here as soon there is progress on this.

**Comment by Time0o** [comment]
@Jayko001 It looks like supabase has addressed this (or not? they mention version `0.1.17` in that issue). And just to clarify, when Postgres 16 support is added, will that fix this error I'm currently seeing?

```
trunk install prometheus_fdw
Using pkglibdir: "/usr/lib/postgresql/16/lib"
Using sharedir: "/usr/share/postgresql/16"
Using Postgres version: 16
Failed to fetch Trunk archive from v1 API: Failed to find an archive for prometheus_fdw vlatest built for PostgreSQL 16
```
